packages/lingua-franca/src/components/layout/SiteFooter.tsx

```diff
@@ -48,6 +48,10 @@ const communityLinks = [
     title: "Web Repo",
     url: "https://github.com/lf-lang/website-lingua-franca",
   },
+  {
+    title: "Slack",
+    url: "https://join.slack.com/t/lf-community/shared_invite/zt-1b4egenxd-Dz~uG5Ps6ce71pgJG02cjw",
```

Comment:
Don't these kinds of invite URLs have a limited life span?

Reply:
Yes, but this link, specifically, is set to never expire.